app/register/controller.ts

```diff
@@ -73,7 +73,7 @@ export default class Register extends Controller.extend(registerQueryParams.Mixi
 
     setup({ queryParams }: { queryParams: RegisterQueryParams }) {
         if (queryParams.campaign) {
-            const matches = queryParams.campaign.match(/^(.*)-(.*)$/);
+            const matches = queryParams.campaign.match(/^(.*)-(.*)|(.*)$/);
             if (matches && matches.length > 2) {
```

Comment:
Hmm, I think this condition will now always be true, which is not what we want. Instead, I think we should keep the regex the same and set the signUpCampaign outside this condition. Is these any reason *not* to do that?